abrt.spec.in

```diff
@@ -23,13 +23,20 @@
 %define have_kexec_tools 1
 %endif
 
-%if 0%{?rhel} >= 6
-    %define desktopvendor redhat
-%else
-    %if 0%{?suse_version}
-        %define desktopvendor suse
+# rpmbuild --define 'desktopvendor mystring'
+%if "x%{desktopvendor}" == 'x'
```

Comment:
Log says:
```
error: parse error in expression
error: /tmp/abrt/abrt.spec:27: bad %if condition
Makefile:1192: recipe for target 'srpm' failed
```

The error occurred because of the quotation marks around 'x'. Use "x" instead. This worked for me.